reference_submissions/librispeech_conformer/librispeech_pytorch/submission.py

```diff
@@ -89,6 +88,8 @@ def update_params(workload: spec.Workload,
     g['lr'] = get_learning_rate(global_step, hyperparameters)
   torch.nn.utils.clip_grad_norm_(current_model.parameters(), max_norm=grad_clip)
   optimizer_state.step()
+  if global_step % 100 == 0:
+    print(f"{global_step}) {train_ctc_loss.item()}")
```

Comment:
I would delete this for now, we can add back in some more logging later as part of the unified logger utils